ratetheapp/src/main/java/uk/co/brightec/ratetheapp/RateTheApp.java

```diff
@@ -243,4 +245,8 @@ public void onRatingChanged(RatingBar ratingBar, final float rating, boolean fro
             }
         }
     };
+
```

Comment:
Going forward, we should add some JavaDoc for any public methods we change/add..